Formula/kubespy.rb

```diff
@@ -16,16 +16,13 @@ class Kubespy < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "aa4dd3522e26e234773a2cdd5ba837ccadcbe4babb7f5889faf8c11dda1bb10d"
   end
 
-  # Bump to 1.18 on the next release, if possible.
-  depends_on "go@1.17" => :build
+  depends_on "go" => :build
 
   def install
     system "go", "build", *std_go_args(ldflags: "-X github.com/pulumi/kubespy/version.Version=#{version}")
   end
 
   test do
     assert_match version.to_s, shell_output("#{bin}/kubespy version")
-
-    assert_match "Unable to read kubectl config", shell_output("#{bin}/kubespy status v1 Pod nginx 2>&1", 1)
```

Comment:
If we are removing this test, then it needs to be replaced with another one, since `--version` tests alone are not sufficient.